src/TLuaInterpreter.cpp

```diff
@@ -11335,7 +11335,7 @@ void TLuaInterpreter::parseJSON(QString& key, const QString& string_data, const
         return;
     }
     int i = 0;
-    for (; i < tokenList.size() - 1; i++) {
+    for (int total = tokenList.size() - 1; i < total; ++i) {
```

Comment:
Does `i` need to be outside of the loop - it seems it may not need to be - although the other `for (...)` on line 11387 will need it's own loop-variable.

Reply:
It does. It's accessed in https://github.com/Mudlet/Mudlet/pull/1754/files#diff-f00fd2abd4401df95552d337ce6c8b32R11359

Reply:
Ok, yeah - my observation was too blinkered. so is it good for you @keneanung - it needs you approval 'cos I cannot speak for the **lua-interface** side of things...